Sentry.Xamarin.Forms/SentryXamarin.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Sentry.Xamarin.Forms;
+using Sentry.Xamarin.Forms.Internals;
+using System;
+
+namespace Sentry
+{
+    /// <summary>
+    /// Sentry Xamarin SDK entrypoint.
+    /// </summary>
+    /// <remarks>
+    /// This is a facade to the SDK instance that also Initializes Sentry .NET SDK.
+    /// use SentrySdk for additional operations (like capturing exceptions, messages,...).
+    /// </remarks>
+    public static class SentryXamarin
+    {
+        internal static SentryXamarinOptions Options { get; set; }
+
+        /// <summary>
+        /// Initializes the SDK with an optional configuration options callback.
+        /// </summary>
+        /// <param name="configureOptions">The configure options.</param>
+        public static void Init(Action<SentryOptions> configureOptions)
```

Comment:
Have SentryXamarinOptions inherit from SentryOptions and do this:

```suggestion
        public static void Init(Action<SentryXamarinOptions> configureOptions)
```